articles/resources/relnotes.md

```diff
@@ -17,6 +17,15 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version X.XX.XXXXXXXX
+
+*Release date: October 26th, 2020*
+
+This release contains the following:
+
+- Improved IQ# debug user experience, to help developers better understand the inner workings of their quantum applications, and more easily identify logic errors. Created visualizations of a particular execution path of a quantum application, as well as the state at a point during the program execution.
```

Comment:
I don't know whether we're ready to announce the debugging and visualization features in the release notes until they are more feature-complete. @efratshabtai @bromeg for additional opinions.

Reply:
@rmshaffer  , since Kitty is OOF, we should make a call about keeping or not this feature in the release notes. What is your recommendation?

Reply:
For the time being, we're completing this PR as is.
@rmshaffer , if you think we should omit this line, please open another PR, and we'll update the docs.

Reply:
This line should not be here. I'll open a new PR.